File: Test/Test-IO/node/TestIO.js
var fs = require('fs');
var content = fs.readFileSync("../DumpFile/dump.txt", 'utf-8');
content = content.split('\n');
for (var i = 0; i < content.length; i++) {
  content[i] = content[i].replace("Tellus", "Terra");
  content[i] = content[i].replace("tellus", "terra");
}
fs.writeFileSync("res.txt", content.join('\n'));

File: Test/Test-IO/python/TestIO.py
#!/usr/bin/python

import timeit, sys, io

def wrapper(func, *args, **kwargs):
    def wrapped():
        return func(*args, **kwargs)
    return wrapped

def start(file, outfile):
    input = open(file, 'r')
    output = open(outfile, 'w')

    line = input.read()
    arr = line.splitlines()
    for i in range(len(arr)):
        arr[i] = arr[i].replace('Tellus', 'Terra')
        arr[i] = arr[i].replace('tellus', 'terra')
        output.write(arr[i] + "\n")

def main(argv):
    file = 'dump.txt'
    output = 'res.txt'
    for i in range(len(argv)):
        if argv[i] == '-f':
            i = i + 1
            file = argv[i]
        elif argv[i] == '-o':
            i = i + 1
            output = argv[i]

    wrapped = wrapper(start, file, output)

    sys.exit(0)

if __name__ == '__main__':main(sys.argv[1:])
